app/views/pages/scholarships.html.erb

```diff
@@ -5,8 +5,8 @@
         <p class="lead">At Operation Code, we partner with both code schools and industry leaders to make transitioning to software development less painful and accessible for our nation's finest and their families.</p>
       </div>
 
-      <div class="center">
-        <iframe src="https://player.vimeo.com/video/125189545" width="700" height="481" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+      <div class="video-responsive">
```

Comment:
I thought we were aiming to use the same class for all the videos on the site - either  `.video-responsive` or `.video-container` for all of them?